data/mods/Aftershock/items/cast_spell_items.json

```diff
@@ -67,7 +67,7 @@
     "subtypes": [ "TOOL", "ARMOR" ],
     "copy-from": "dimensional_anchor",
     "name": { "str": "recall rig" },
-    "description": "A complex antennae assembly worn around the upper torso that allows its user to home and teleport towards previously configured teleport pads.  Although relatively functional, it's much inferior to the translocation systems of old, and a successful teleport requires standing perfectly still for several minutes.",
+    "description": "A complex antennae assembly worn around the upper torso that allows it's user to teleport towards previously configured teleport pads.  Although relatively functional, it's much inferior to the translocation systems of old, and a successful teleport requires standing perfectly still for several minutes.",
```

Comment:
```suggestion
    "description": "A complex antennae assembly worn around the upper torso that allows its user to teleport towards previously configured teleport pads.  Although relatively functional, it's much inferior to the translocation systems of old, and a successful teleport requires standing perfectly still for several minutes.",
```

Reply:
I'm gonna split the difference on this one and say this makes more sense but you can edit as you think is best. I like it removing the dangling home.